ui/raidboss/data/04-sb/raid/o2s.js

```diff
@@ -244,29 +292,37 @@ export default {
         },
       },
     },
+    {
+      id: 'O2S Maniacal Probe You',
+      netRegex: NetRegexes.headMarker({ id: '0005 ' }),
+      condition: Conditions.targetIsYou(),
+      alertText: (_data, _matches, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Probe on YOU',
+        },
+      },
+    },
     {
       id: 'O2S Unstable Gravity',
       netRegex: NetRegexes.gainsEffect({ effectId: '550' }),
       condition: Conditions.targetIsYou(),
-      delaySeconds: 9,
-      alarmText: (_data, _matches, output) => output.elevateOutsideStack(),
-      tts: (_data, _matches, output) => output.floatForBomb(),
+      alarmText: (_data, _matches, output) => output.text(),
       outputStrings: {
-        elevateOutsideStack: {
-          en: 'Unstable Gravity: Elevate and outside stack',
-          de: 'Schwerkraftschwankung: Schweben und außen stacken',
-          fr: 'Gravité instable : Lévitez et packez-vous à l\'extérieur',
-          ja: 'グラビティバースト: 浮上集合',
-          cn: '升起并分摊',
-          ko: '중력 폭탄: 공중부양 및 쉐어',
+        text: {
+          en: 'Elevate (Unstable Gravity)',
```

Comment:
```suggestion
          en: 'Elevate (Unstable Gravity)',
          ko: '공중부양하기 (중력 폭발)',
```

Reply:
Same with the other one, I wonder if this should be `Unstable Gravity on YOU (Elevate)`

Reply:
I wanted to lead with the verb.  I could say `Elevate (Unstable Gravity on YOU)` if you think that's better?

Reply:
I don't think that's necessary; I think it's reasonable as is.